group_vars/all/main.yml

```diff
@@ -46,6 +46,12 @@ trashable_env_types:
   - "development"
   - "feature"
 
+# To be use with edxapp_sql_dump_url in edxapp config
+# This is useful when migration become too long to execute. We can
+# provide a sql dump of the schema to then load it instead of waiting
+# for migrate to be done (works only with trashable env)
+enable_sql_dump_loading: false
```

Comment:
If it defaults to `false` you are introducing a BC.

Reply:
Right good point